### PR TITLE
Upgrade to JSCS 3

### PR DESCRIPTION
**Pull request by martndemus**
Needed for: https://github.com/DockYard/ember-suave/pull/120

### CODE REVIEW

**Comment by minichate** [comment]
Thanks! Let me know if I can help somehow

**Comment by martndemus** [comment]
@minichate Somehow it can't find the cst parser when running the tests on CI.

I'm trying to fix it, but can't find what's up.

File: lib/rules/disallow-privateregistryproperty.js [hunk 1]
```diff
@@ -30,30 +30,18 @@ module.exports.prototype.check = function(file, errors) {
...
-        line: node.parentNode.loc.start.line,
-        column: node.parentNode.loc.start.column,
-        additional: node
+        element: node
       });
     }
   });
 };
-
-module.exports.prototype._fix = function(file, error) {
```
Comment (martndemus) [inline lib/rules/disallow-privateregistryproperty.js]: This fix still needs to be refactored to 3.0 code

File: package.json [hunk 2]
```diff
@@ -24,12 +24,12 @@
   "dependencies": {},
   "devDependencies": {
     "chai": "^3.4.1",
-    "chai-subset": "^1.1.0",
+    "chai-subset": "^1.2.2",
     "coveralls": "^2.11.4",
-    "esprima": "^2.7.0",
+    "cst": "^0.3.0",
```
Comment (brzpegasus) [inline package.json]: Some rules in `lib` explicitly require the `cst` module so it needs to be a dependency, not dev dependency.

**Pull request merged**